src/ResourceManager/MachineLearning/Commands.MachineLearning.Test/ScenarioTests/Common.ps1

```diff
@@ -54,21 +54,6 @@ Gets the default location for a provider
 #>
 function Get-ProviderLocation($providerNamespace, $resourceType)
 {
-    if ([Microsoft.Azure.Test.HttpRecorder.HttpMockServer]::Mode -ne `
```

Comment:
Trying to make the test execution predictable and not impacted by other cmdlet which not related to the test target.